requirements.txt

```diff
@@ -13,4 +13,6 @@ safe-eth-py==4.1.3
 slackclient==2.9.4
 web3>=5.30.0
 SQLAlchemy>=1.4.41
+sqlalchemy-stubs==0.4
 pandas==1.5.0
+pandas-stubs==1.5.1.221024
```

Comment:
#TIL There exists a pandas-stubs repo. Quite interesting that they've separated this. Cool you found this!